src/shared/ui/PlanetImage/index.tsx

```diff
@@ -0,0 +1,2 @@
+import PlanetImage from "./PlanetImage";
+export default PlanetImage;
```

Comment:
if you get extra files in the folder? 
I suggest using aggregate export example: export * from './fileName'.